javascript/chronometer.js

```diff
@@ -1,34 +1,46 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1;
+    }, 1000);
+    if(callback){
+      setInterval((callback), 1000);
+    }
   }
```

Comment:
Now you have two separate intervals running! Which is not exactly correct, we want to stick to just one interval, and check for callback inside it 

```
  start(callback) {
    this.intervalId = setInterval(() => {
      this.currentTime++;
      if (callback) {
        callback();
      }
    }, 1000);
```